sys/fuchsia/fidl_process.txt

```diff
@@ -1,13 +1,90 @@
-# Copyright 2018 syzkaller project authors. All rights reserved.
-# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+# Copyright 2018 The Fuchsia Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+#
+# WARNING: This file is machine generated by fidlgen.
 
-include <zircon/syscalls.h>
+fuchsia_process_LauncherLaunchRequest {
+	txid		int32[0]
+	reserved	int32[0]
+	flags		int32[0]
+	ordinal		int32[1]
```

Comment:
All of these 4 fields need to be `const[VAL, int32]`. Const is stronger requirement than int, and that's we generally use for cases when there is only 1 fixed valid value.

Reply:
Done.